thirdparty/fetch-thirdparty-deps.sh

```diff
@@ -42,6 +42,7 @@ if [ ! -f SharpFont.dll ]; then
 	nuget install SharpFont -Version 2.5.0.1
 	cp ./SharpFont.2.5.0.1/lib/net20/SharpFont* .
 	cp ./SharpFont.2.5.0.1/Content/SharpFont.dll.config .
+	sed -i '/osx/s@\(dll="\)[^"]*\(" />\)@\1/Library/Frameworks/Mono.framework/Libraries/libfreetype.6.dylib\2@' SharpFont.dll.config
```

Comment:
This usage of `-i` is a nonstandard GNUism, so it doesn't work with OSX's BSD sed.